Tests/Tests/STPAddSourceViewControllerTest.m

```diff
@@ -216,13 +238,13 @@ - (void)testCard_nextWithCreateSourceSuccessAndDidCreateSourceSuccess {
 #pragma mark - SEPA Debit
 
 - (void)testSEPA_nextWithCreateSourceError {
-    STPAddSourceViewController *sut = [self buildAddSourceViewControllerWithType:STPSourceTypeSEPADebit];
+    STPUserInformation *info = [self buildUserInfoForSEPATests];
+    STPAddSourceViewController *sut = [self buildAddSourceViewControllerWithType:STPSourceTypeSEPADebit
+                                                                        userInfo:info];
     id mockAPIClient = OCMClassMock([STPAPIClient class]);
     sut.apiClient = mockAPIClient;
-    sut.ibanCell.contents = @"GB82WEST12345698765432";
+    sut.ibanCell.contents = @"DE89370400440532013000";
```

Comment:
had to change these to IBANs to match the `sepaAddress` fixture, which is a German address (this line was changing the expected address's country to "GB")